descartes_trajectory/test/cart_trajectory_pt.cpp

```diff
@@ -1,160 +0,0 @@
-/*
```

Comment:
I deleted an extremely long running test (>1 minute), `getPoses` that counted angle discretization. A real pain to work with while doing this work. I will be-grudgingly put it back.